src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -117,6 +117,15 @@
     int getHighestYAt(int x, int z);
 
     /**
+     * Returns the y level that precipitation ends falling in the given column.
+     *
+     * <p>A value is still returned for columns in biomes which do not receive precipitation.</p>
+     *
+     * @return the y level that precipitation ends
+     */
+    int getPrecipitationLevel(int x, int z);
```

Comment:
Add another method which accepts `Vector2i` - the new method can be `default`-implemented and call this method.